Documentation/Introduction/Index.rst

```diff
@@ -54,15 +54,15 @@ to existing tables and add new tables. Several required extensions that are
 always loaded, already deliver some TCA files in their
 :file:`Configuration/TCA` directories.
 
-Most importantly, the extension "core" comes with a definition of pages,
+Most importantly, the extension "cms-core" comes with a definition of pages,
 be_users and further tables needed by the whole system.
-The extension "frontend" comes with the tables tt_content, fe_users, sys_template and more.
-See the directories :file:`typo3/sysext/core/Configuration/TCA/` and
-:file:`typo3/sysext/frontend/Configuration/TCA/` for the complete list of the TYPO3 CMS tables.
+The extension "cms-frontend" comes with the tables tt_content, fe_users, sys_template and more.
+See the directories :file:`vendor/typo3/cms-core/Configuration/TCA/` and
+:file:`vendor/typo3/cms-frontend/Configuration/TCA/` for the complete list of the TYPO3 CMS tables.
 
 The TCA definition of a new table with the name "database-table-name" must be done in the
 extension directory :file:`Configuration/TCA/` with :file:`database-table-name.php` as filename.
-An example is :file:`EXT:sys_note/Configuration/TCA/sys_note.php` for table "sys_note". This file will be
+An example is :file:`EXT:cms-sys_note/Configuration/TCA/sys_note.php` for table "sys_note". This file will be
```

Comment:
```suggestion
An example is :file:`EXT:sys_note/Configuration/TCA/sys_note.php` for table "sys_note". This file will be
```
The extension is still called sys_note

Reply:
Ah, this was not clear to me. I thought that the name of the extension and the path in the project is the same.

Reply:
@Alagts Your assumption is partly correct - however, only for a non composer (lagacy) installation (`typo3conf/ext/<extension-key>` => `EXT:<extension-key>`

That differs from a composer package name (`vendor/package-name` ) which needs to hav a `extra` => `typo3/cms` => `extension-key` set to define the extension name. Since v12 extension are no longer installed to `typo3conf/ext/<extension-key>` and left in `vendor/<vendor>/<package-name>` (instead of the extension key name). Assets are not symliked by name, but by hash etc.

;)